core/runtimes/FiberRuntime/index.js

```diff
@@ -197,17 +186,19 @@ FiberRuntime.prototype._procState = function (proc) {
 // Fibre running scheduler
 FiberRuntime.prototype._actualRun = function () {
   this._queue = this._genInitialQueue();
-  function setPortRuntime(port) {
```

Comment:
This was the killer. `this` has a special meaning within a function. It binds to what's called the function's context whenever the function is called.

Using the `bind` method allows you to make that binding explicit and permanent.